### ISSUE TITLE
Accessibility: WCAG violation by using "Fullscreen mode" as default in post editor

### ISSUE THREAD

**Issue by burnuser** [WordPress/gutenberg#47112]
### Description

Long lasting problem since WP 5.4
As reported in core https://core.trac.wordpress.org/ticket/57450 and advised to report it here.
(And please don't play "not right located here" tennis!)

### Step-by-step reproduction instructions

After login, a new WordPress user is confronted with consistent administration screens:
A) Toolbar at the top of the screen
B) Main Navigation menu at the left side of the screen

Making basic administrative adjustments works all in this - everywhere the same - environment.

But after adding a new post or page, the common options of toolbar and Main Navigation are gone!
Panic!
Don't know how to get out of this damn post editor!
Or even Log out from WordPress!
Panic! Confusion! Frustration!

That's an really awful first (and for some users also last) experience for starting with WordPress!
(And - as an experienced WordPress trainer - I'm stating this as a many times proven fact!)
Never ever should a user interface lock anybody in a regular editor screen without common/obvious way out!
(And no, clicking on the WordPress Icon to get out there has - in my entire experience as trainer - never been discovered by a trainee without separate explanation!)

I know, the decision to set "Fullscreen mode" as default in post editor is not new, but it is never to late, to make something better!

By the way:
1.) "Collapse menu" in the sidebar gives many space => No need for "Fullscreen mode" anyway!
2.) The missing Toobar at the top in "Fullscreen mode" breaks also the workflow of viewing and adding post - or jumping to administrative functions - with one click!

And if terrible usability and user-experience is not enough to revert this (non working) change, maybe compliance with the law - like mandatory WCAG in Europe - is a sufficient incentive.

Because this unnecessary change in user-interface is a direct violation of the "Web Content Accessibility Guidelines"
=> "Make Web pages appear and operate in predictable ways"
https://www.w3.org/TR/WCAG21/#consistent-navigation

Which also makes clear, that a change to "Fullscreen mode" by the user (= own decision) is OK.

### Environment info

WP 5.4-6.1.1

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes
